.circleci/config.yml

```diff
@@ -1815,56 +1815,78 @@ workflows:
             branches:
               only: /(^master$)/
 
-#   release-to-service-providers-cron:
-#     jobs:
-#       - release-week
-#       - release-scripts:
-#           name: release-to-service-providers
-#           target: release-to-service-providers
-#           context:
-#             - slack-secrets
-#             - gh-tokens.releases
-#             - discord
-#             - governance
-#     triggers:
-#       - schedule:
-#           cron: '0 15 * * 1'
-#           filters:
-#             branches:
-#               only: /(^master$)/
-
-#   evaluate-proposal-cron:
-#     jobs:
-#       - release-week
-#       - release-scripts:
-#           name: evaluate-proposal
-#           target: evaluate-proposal
-#           context:
-#             - slack-secrets
-#             - governance
-#     triggers:
-#       - schedule:
-#           cron: '0 19 * * 5'
-#           filters:
-#             branches:
-#               only: /(^master$)/
-
-#   cut-release-cron:
-#     jobs:
-#       - release-week
-#       - release-scripts:
-#           name: cut-release
-#           target: cut-release
-#           context:
-#             - slack-secrets
-#             - gh-tokens.releases
-#       - generate-release-checklist
-#     triggers:
-#       - schedule:
-#           cron: '0 7 * * 6'
-#           filters:
-#             branches:
-#               only: /(^master$)/
+  joaquins-cron-test:
```

Comment:
Was this meant to be uncommented?

Reply:
Nope, this is the test for today, before I uncomment the other code again.

Reply:
Nah, it's just a cronjob name that I'll be removing in a few more minutes.